ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -1520,6 +1520,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-stack-ipi-proxy
+- as: azure-stack-ipi-proxy-f14-disasterrecovery
```

Comment:
My last comment, is it possible to use `f28`, because azure stack is a small deployment, can not bear many concurrent testing.

Reply:
Sure, agree.

Reply:
But this is a new profile for 4.16, can we consider doing that after 4.16 is released?

Reply:
okay